components/dashboard/src/components/Menu.tsx

```diff
@@ -4,83 +4,191 @@
-                        separator: true
-                    },
-                    {
-                        title: 'Settings',
-                        link: '/settings',
-                        separator: true
-                    },
-                    {
-                        title: 'Logout',
-                        href: gitpodHostUrl.asApiLogout().toString()
-                    },
-                    ]}>
-                        <img className="rounded-full w-6 h-6"
-                            src={user?.avatarUrl || ''} alt={user?.name || 'Anonymous'} />
-                    </ContextMenu>
+    const userFullName = user?.fullName || user?.name || '...';
+    const showTeamsUI = !!user?.rolesOrPermissions?.includes('teams-and-projects');
+    const team = getCurrentTeam(location, teams);
+
+    const [ teamMembers, setTeamMembers ] = useState<Record<string, TeamMemberInfo[]>>({});
+    useEffect(() => {
+        if (!showTeamsUI || !teams) {
+            return;
+        }
+        (async () => {
+            const members: Record<string, TeamMemberInfo[]> = {};
+            await Promise.all(teams.map(async (team) => {
+                const infos = await getGitpodService().server.getTeamMembers(team.id);
+                members[team.id] = infos;
+            }));
+            setTeamMembers(members);
+        })();
+    }, [ teams ]);
+
+    const leftMenu = (!!team
+        ? [
+            {
+                title: 'Projects',
+                link: `/${team.slug}/projects`,
+                alternatives: [`/${team.slug}`]
+            },
+            {
+                title: 'Members',
+                link: `/${team.slug}/members`
+            }
+        ]
+        : [
+            {
+                title: 'Workspaces',
+                link: '/workspaces',
+                alternatives: ['/']
+            },
+            {
+                title: 'Settings',
+                link: '/settings',
+                alternatives: settingsMenu.flatMap(e => e.link)
+            }
+        ]
+    );
+    const rightMenu = [
+        ...(user?.rolesOrPermissions?.includes('admin') ? [{
+            title: 'Admin',
+            link: '/admin',
+            alternatives: adminMenu.flatMap(e => e.link)
+        }] : []),
+        {
+            title: 'Docs',
+            link: 'https://www.gitpod.io/docs/',
+        },
+        {
+            title: 'Community',
+            link: 'https://community.gitpod.io/',
+        }
+    ];
+
+    return <>
+        <header className="lg:px-28 px-10 flex flex-col pt-4 space-y-4">
+            <div className="flex">
+                <div className="flex justify-between items-center pr-3">
+                    <Link to="/">
+                        <img src={gitpodIcon} className="h-6" />
+                    </Link>
+                    <div className="ml-2 text-base">
+                        {showTeamsUI
+                            ? <ContextMenu classes="w-64 left-0" menuEntries={[
```

Comment:
**question**: Could we `bg-gray-100` on hover and also use this background for the currently active team? 💭

Reply:
Now tracked in https://github.com/gitpod-io/gitpod/issues/4471